File: Frontend/src/app/models/Product.ts
export class Product{

    constructor(
        public name: string,
        public description: string,
        public price:number,
        public storeID:string,
        public categoria:string,
        public amount:number,
        public id?:string,
    ){}

}

export enum Categoria {
  OPCION1 = 'SALUD',
  OPCION2 = 'CONSTRUCCION'
}

File: Backend/src/main/java/com/udla/ingweb/backend/Entity/Product.java
package com.udla.ingweb.backend.Entity;

import lombok.Getter;
import lombok.RequiredArgsConstructor;
import lombok.Setter;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;


@Document(collection =  "products")
@Getter
@Setter
@RequiredArgsConstructor
public class Product {
    @Id
    private String id;
    private String name;
    private String description;
    private String categoria;
    private float price;
    private String storeID;
    private int amount;
}
